docker-compose.dev.yml

```
@@ -185,8 +190,83 @@ services:
       graph-db:
         condition: service_healthy
 
```

We can declare our separate bh-api-sso defintion here

```suggestion
  bh-api-sso:
    <<: *bh-api
    profiles:
      - sso
    links:
      - authentik:authentik.localhost
```